lib/user/pages/user_settings_page.dart

```diff
@@ -163,25 +163,28 @@ class _UserSettingsPageState extends State<UserSettingsPage> {
                         onToggle: (bool value) => {context.read<UserSettingsBloc>().add(UpdateUserSettingsEvent(showBotAccounts: value))},
                       ),
                     ),
-                    Padding(
-                      padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
-                      child: Text(l10n.filters, style: theme.textTheme.titleMedium),
-                    ),
                     if (LemmyClient.instance.supportsFeature(LemmyFeature.blockInstance)) ...[
-                      UserSettingTopic(
-                        title: l10n.blockedInstances,
-                        trailing: IconButton(
-                          icon: Icon(
-                            Icons.add_rounded,
-                            semanticLabel: l10n.add,
-                          ),
-                          onPressed: () => showInstanceInputDialog(
-                            context,
-                            title: l10n.blockInstance,
-                            onInstanceSelected: (instance) {
-                              context.read<UserSettingsBloc>().add(UnblockInstanceEvent(instanceId: instance.id, unblock: false));
-                            },
-                          ),
+                      Padding(
+                        padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
+                        child: Row(

```

Comment:
The only tiny problem with these changes is that the plus button (to add a new block) and the x button (to delete a block) are no longer aligned. It's very minor though. 😊

Reply:
I think this should be okay for now! I do think the plus button now lines up with the ToggleOptions so maybe its just a matter of adjusting the x button to be closer to the right

Reply:
> maybe its just a matter of adjusting the x button to be closer to the right

Agreed!